src/main/scala/com/monsanto/arch/cloudformation/model/simple/Builders.scala

```diff
@@ -244,6 +181,18 @@ trait Subnet extends AvailabilityZone with Outputs {
 
     f(sub) ++ sub.andOutput(name, name)
   }
+
+  def nat(routeTables: Seq[`AWS::EC2::RouteTable`])(implicit s: `AWS::EC2::Subnet`) = {
```

Comment:
Are there other places where subnets are implicit?  If that's an established convention, I suppose it's fine.  However, it seems better to me as an explicit parameter.